### PR TITLE
I added to my README file a Markdown table with links to my social sites, added the Task Section, the Research Section and the blog post describing things I’ve learned recently.

### PR DESCRIPTION
**Pull request by kofodym**
- I added to my README file a Markdown table with links to my social sites.
- I added the `## Tasks` section to my README file.
- Checked all tasks in "Getting started with Outreachy".
- Checked all tasks in "Open source contributions".
- Checked all tasks in "Your contribution period with Wagtail".
- I added the Research section containing the links on the Researches made on Wagtail and the Accessibility Features Documentation
- I added a new section, Blog post describing things I’ve learned recently.

### CODE REVIEW

File: 2023/Cole Kofoworola/README.md [hunk 1]
```diff
@@ -7,3 +7,14 @@ I started learning front-end development during my career break during childbirt
...
 
 This is my first time participating in Outreachy. I read about Open Source sometime this year and made 2 contributions to a friend's project and it felt like I made a significant impact. It was amazing. I am happy to work with everyone.
+
+
+##Social Links
+
+| **Social Site**     | **Link**                                                        |
+| :------------------ | :-------------------------------------------------------------- |
+| My GitHub profile   | [kofodym](https://github.com/kofodym)                           |
+| My personal blog    | [kofocole](https://www.kofocole.hashnode.dev)                   |
```
Comment (thibaudcolas) [inline 2023/Cole Kofoworola/README.md:17]: Probably https://kofocole.hashnode.dev/? I get an error with `www.`

Comment (kofodym) [reply 2023/Cole Kofoworola/README.md]: Thank you @thibaudcolas . I also noticed my linkedin link was bad. I have adjusted them appropriately and sent a PR

File: 2023/Cole Kofoworola/README.md [hunk 2]
```diff
@@ -7,3 +7,120 @@ I started learning front-end development during my career break during childbirt
...
+- [Why 96.3% of the Web Fails at Accessibility - GAAD 2023](https://thib.me/why-96-3-percent-of-the-web-fails-at-accessibility-gaad-2023)
+- [Everything you need to know about WCAG 2.2 Accessibility Standards](https://reciteme.com/news/wcag-2-2-accessibility-standards/)
+- [What is accessibility?](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility)
+- [Why Web Accessibility Is Important and How You Can Accomplish It](https://medium.com/fbdevclagos/why-web-accessibility-is-important-and-how-you-can-accomplish-it-4f59fda7859c)
+- [How To Do Website Accessibility The Right Way](https://medium.com/@adasitecompliance/how-to-do-website-accessibility-the-right-way-9ee8b395e9a)
+- [Checking a website for accessibility](https://www.washington.edu/accesstech/websites/)
+
+## Blog post describing things I’ve learned recently
+
+- [What I have learned recently](https://kofocole.hashnode.dev/things-i-have-leaned-recently-about-contributing-to-opensource)
```
Comment (thibaudcolas) [inline 2023/Cole Kofoworola/README.md:126]: I like your blog post but I would generally recommend avoiding images of text. They’re very poor for accessibility. You also used the wrong alt text, the image and its alt text say something different.

**Review by thibaudcolas — APPROVED** [review]

File: 2023/Cole Kofoworola/README.md (hunk 2 above)
Comment (kofodym) [reply 2023/Cole Kofoworola/README.md:126]: Thank you so much. I have tried tracing the alt text with an accessibility checker, but it appears hashnode.com cover image has no option to add an alt text. I have now removed the cover image and added an image in the body of my blog instead. I have also added appropriate alt text describing the image.

**Pull request merged**